demo/CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# Changelog
+
+All notable changes to the `demo` component will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [1.1] - 2021-01-11
+
+-This release was authored by Lakshay Sethi.
+-Changed: A single workflow divided into 3 seperate worklows, for an easier time for the user.
```

Comment:
Typically refactoring is the lowest priority and should only be done after high priority tasks (test writing, making things work, fixing bugs, etc.) are finished.